usage.py

```diff
@@ -65,6 +65,7 @@
                         dict(label='option 3', value='3'),
                     ],
                     valueSelected='1',
+                    row=True,
```

Comment:
This works, but it takes up a lot of room. Let's keep it as a column of buttons to fit the layout.